### ISSUE TITLE
Create some "persona" 

### ISSUE THREAD

**Issue by bjoernbroetz** [halo-db/storymap#1]
Wir erfinden ein paar virtuelle Leute, die unsere Usergruppen repräsentieren. Z.B.:
---------------------------------------------
![20210118_Steffi](https://user-images.githubusercontent.com/25008190/104895607-632f7680-5976-11eb-8c21-b066f8cde9eb.jpeg)

Steffi
PhD Student am AWI
24 Jahre
Arbeitet mit Windows / Excel / IGOR
möchte Python lernen
Nimmt teil an HALO-Kampagne
Möchte Daten mit Kollegen austauschen
---------------------------------------------

**Comment by bjoernbroetz** [comment]
-------------------------------------------------------
![20210118_Roland](https://user-images.githubusercontent.com/25008190/104895915-c0c3c300-5976-11eb-8856-92904eb22d23.jpeg)

Roland
PI eines Messgerätes auf HALO
Senior Wissenschaftler
53 Jahre
Windows/Linux
IDL, Matlab, FORTRAN
-------------------------------------------------

**Comment by d70-t** [comment]
![Francesca](https://user-images.githubusercontent.com/6574622/105030072-8714ba00-5a53-11eb-8f0f-0d7ddbd3f68c.png)

Francesca just received a Major review on her first own paper. While the review was in positive in general, it requires to reprocess almost all of her input data. Unfortunately, the harddrive of her computer ceased to work, so the carefully assembled folder structure for the input data is lost. So this is a very unfortunate moment to ask for her mood.
* Just finished her master's thesis
* Good knowledge in Python / Latex, uses git for code as someone said it's cool

**Comment by d70-t** [comment]
![Danny](https://user-images.githubusercontent.com/6574622/105030230-cb07bf00-5a53-11eb-96bf-379266e7af9c.png)

Danny builds instruments and is good in electronics and software development. He doesn't like policies very much, but is a fan of open data and robust technical solutions. But as as producer of datasets, using the HALO aircraft as an instrument platform, he has to comply to the data policies, which he prefers to do with creativity.
* Permanent position as observational scientist
* Knows everything of Rust, C++, LabView, Python, Julia, Micropocessors, digital and analog electronics.
* Regularly produces datasets in the oder of 100 TB per year which need to be published somewhere
* His institution doesn't have the required infrastructure to serve all the data reliably to the public

**Comment by d70-t** [comment]
![David](https://user-images.githubusercontent.com/6574622/105045438-33609b80-5a68-11eb-96fb-9b29b6c88da6.png)

David likes to dig into the data. He is very good in combining measurements from different instruments and creates high-quality visualizations, which make it easy to follow his new ideas. He likes to spread his ideas and to collaborate with other people from all over the world. He often exchanges scripts or notebooks with collaborators, but more often than not, it takes hours or even days to get those running on the computers of his friends, because the access to his source datasets is tricky and the local folder structures are different.
* Started with atmospheric sciences because his uncle (who is a farmer) told him that the weather is very different to when he was young and it becomes more and more difficult to grow crops.
* Is now in his second post doc
* Has many more ideas than he can investigate on his own
* Knows Python, Fortran and C
* Uses Windows

**Comment by d70-t** [comment]
![Selma](https://user-images.githubusercontent.com/6574622/105047156-42e0e400-5a6a-11eb-9204-897157a18a7a.png)

Selma studied comunication sciences, which she finished seven years ago with a master's degree. After working for five years in public relations within a large oil company, she and a friend decided that they need to do something different. That's why the founded a non profit organization "f-earth-t hand climate" (1hc) to educate people on climate change on an open access basis. The key idea of 1hc is that people will better understand how the earth's climate behaves if they can understand and interpret measurement data themselves. Selma and her team build interactive visualizations of routinely taken data but want to enhance their collection of articles with data from field campaigns. In support of their main goal, all data analysis is done in a variant of [NodeRED](https://nodered.org) such that readers can easily follow the author's arguments and als try out other interpretations of the data.

* Knows how to communicate things in a way that people are convinced
* Believes that oil is not the solution and likes to see cities with green alleys in stead of car parking spaces
* Just started to learn JavaScript
* Knows people who are good in JavaScript, wasm, HTML, CSS and WebGL
* "The browser is my operating system"

**Comment by d70-t** [comment]
![Friedrich](https://user-images.githubusercontent.com/6574622/105050214-d7991100-5a6d-11eb-97d0-bbcb7ebb4e0e.png)

Friedrich is the guy to ask for the money. Well, technically, that's not true, but practically he's the chief advisor at LFA (the Largest Funding Agency). So if you can convince him, you'll have all the money you need, if you don't then it will be tough. Back when he still worked hands-on on meteorological observations, tabulating temperature readings on a sheet of paper was still not uncommon. He's regularly attending conferences and is well informed about the current frontiers in atmospheric sciences and in the physics of atmospheric measurement techniques. Apart from basic telecommunication needs, network and datacenter infrastructure isn't important for his daily life and he doesn't know a lot about that. Open data is still kind of new to him, but LFA policies change slowly into that direction, so he can't ignore it anymore.

* Knows Word, Excel, Powerpoint and E-Mail
* Likes to be in the loop
* 60 years old

**Comment by d70-t** [comment]
![Jana](https://user-images.githubusercontent.com/6574622/105744925-0287d700-5f3e-11eb-9883-c2e5def55f18.png)
Jana is a sculptor mostly interested in building illuminated art pieces which are placed in the public. She bases her works on interesting scientific datasets which, in her opinion, should be discussed more in the general public. She started by presenting paleontological findings and has already built several own pieces which highlight changes in underwater biology, especially coral reefs using [Generative art](https://en.wikipedia.org/wiki/Generative_art). Currently she is searching for new ideas and became interested in the atmosphere, but hasn't chosen a final topic yet.

* Good in Machining and Electronics
* Mostly programs in [Processing](https://processing.org)

**Comment by d70-t** [comment]
![Rhoda](https://user-images.githubusercontent.com/6574622/105746736-1c2a1e00-5f40-11eb-90f6-d5def98d2644.png)
Rhoda is a university professor since three years. Her group analyzes field campaign data from the HALO aircraft and she's hoping that her new instrument idea will get funded next summer. What she finds frustrating is that it is really hard to get new students up and running with their own analysis projects i.e. for creating a bachelor's thesis. Scripts which run on one computer don't run on another, data is hidden in an external disk of her first student who did a good job but went away to become surf trainer and there also have been cases in which the datasets metadata actually told the opposite of what has actually been inside the file 🤷. As she expects to become a dataset creator as well, she really wants to do better, but doesn't yet know the best solution.

**Comment by kdgo** [comment]
![Mario](https://user-images.githubusercontent.com/25305266/105852279-8e543e80-5fe4-11eb-8e04-b03a0367794d.png)
**Mario** is a **modeller** and always keen to find interesting applications for his simulations. He knows everything  about data formats, programming and shell scripts. For better comparability [simulations should be sampled matching the observations in space and time](https://github.com/halo-db/storymap/issues/28#issue-794143990). However, Mario is not part of all campaign teams from the start. But this does not matter, as long as he can feed the flight tracks into the setup before starting a new simulation, and as long as he gets a chance to offer his results to the corresponding observational crew. Issues:
- Excempt track information from data protocols?  
- automated queries or transfer of latest track data?

**Comment by kdgo** [comment]
![Dimitri](https://user-images.githubusercontent.com/25305266/105853296-d1fb7800-5fe5-11eb-9ca6-e1c5d80ab86f.png)
**Dimitri** takes care about **essential aircraft instrumentation**, e.g. flight track data, measurements of pressure and temperature. His data are needed for almost any scientific data analysis. Particularly the flight track is also important as input for modelling and as part of campaign metadata. Dimitri would like to distribute his data to everyone who is interested, asap. At the same time he would like to stay in control for updates and corrections. Issues:

- Establish special procedures for this kind of data?

**Comment by kdgo** [comment]
![Gislinde](https://user-images.githubusercontent.com/25305266/105857425-92835a80-5fea-11eb-9244-1fce6fbedff5.png)
**Gislinde** is a **senior professor** and has organized several aircraft campaigns already. She wants to spend time on science, but her day is all too often consumed by fighting for money, supervising students, juggling deadlines, problems with instruments and certifications. So data management is not her top priority: It's scientifically worthless and has never required special attention so far. However, she wants keep track of which data are available and she wants to control who can access those. If all of a sudden an idea for a Nature paper pops up, everything needed for that must be available quickly. How was the weather during the flight? Are there any simulations for that case? Are the aerosol data processed already or even useful? Who else has checked out the data and is possibly working on the same idea? What exactly do the journal guidelines about data availability mean and how do I get such an DOI before tomorrow's deadline? Issues:

- ease of use in the context of the needs of busy high-level scientists
- big brother vs privacy?

**Comment by kdgo** [comment]
![Kurt](https://user-images.githubusercontent.com/25305266/105869319-64f0de00-5ff7-11eb-8399-f04471a5de4e.png)
**Kurt** is the **head** of department/institute. Sometimes he needs a place for dumping the data of his current campaign, sometimes he needs some old data. A short request to the in-house data guy saves him time. Also, the term data management seems to come about more often, so it's good to have an eye on that. On the other hand, its just another boring nuisance that never results in a paper. What are the relevant parameters of a database anyway? It has not aquired any third-party funding so far. However, the status quo is affordable and the position also helps with real projects. Issues:

- How to get support from decision makers?
- How to ensure long-term sustainability of the database: daily operation / technical maintenance & small updates/ major overhauls?

**Comment by kdgo** [comment]
![Raj](https://user-images.githubusercontent.com/25305266/105871626-ca45ce80-5ff9-11eb-965c-2ce12417e7ba.png)
**Raj** just **started his PhD** about lightning in the Asian monsoon. His small University at the foothills of the Himalayas is internationally not very well connected, but he has read about this aircraft campaign, which took place some time ago in the UTLS above his head. Do they possibly have NOx data Raj could use? How could he find and get them?

**Comment by kdgo** [comment]
![Li](https://user-images.githubusercontent.com/25305266/105885598-9f637680-6009-11eb-80f4-28c61c0a6b10.png)
**Li** took part in a HALO campaign with an instrument from institute A. It bestowed a PhD on her and she manged to publish a paper about it. The data are in the HALO DB and got a DOI assigned to them. Now she is a postdoc in institute B. Issues:

- **change of affiliation**,
- transfer of access rights?,
- there is a bug in the published data that needs correction

**Comment by kdgo** [comment]
![Heinz-Dietrich](https://user-images.githubusercontent.com/25305266/105888576-6d541380-600d-11eb-8230-07ff1bc7df8b.png)
**Heinz-Dietrich** generated a lot of data during his long career as instrument PI. The more recent of those data even made it into the HALO DB, at least mostly. Now he enjoys his well-deserved **retirement**, probably somewhere in Sweden. When H.-D. retired, he wanted to stay in control of his data, because now he has the time to write all those papers. Perhaps.  At the same time a student would like to correct and then work with his data.  
Issues:
- How to deal with the heritage of people who left the community?
- open access helps, but allowing and tracing data modifications by others is another story

**Comment by kdgo** [comment]
![Rolf](https://user-images.githubusercontent.com/25305266/105894378-83b19d80-6014-11eb-8c26-12e750782106.png)
Rolf is a technician and knows everything about the instrument he operates in the name of [Kurt](https://github.com/halo-db/storymap/issues/1#issuecomment-767641618), his boss. Rolf's experience is essential to transform raw data into something useful - applying a combination of 1990's scripts and propriatary software systems, controlled by selecting choices from menus. Data management and data bases are not his type of thing. He just wants to dump his data with as little effort as possible, perhaps in the name of his boss. Issues:

- Reproducibility: How much information should be kept about intermediate processing steps?
- How do deal with de-facto shared ownership of data? Give more than one person the right to update data? Encourage shared accounts (like e.g. the Ramses account of the flight department)?
